src/apps/material/material.tsx

```diff
@@ -160,112 +157,122 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
   const shouldOpenReviewDisclosure = !!getUrlQueryParam("disclosure");
 
   return (
-    <section className="material-page">
-      <MaterialHeader
-        wid={wid}
-        work={work}
-        selectedManifestations={selectedManifestations}
-        setSelectedManifestations={setSelectedManifestations}
-        selectedPeriodical={selectedPeriodical}
-        selectPeriodicalHandler={setSelectedPeriodical}
-      />
-      <MaterialDescription pid={pid} work={work} />
-      <Disclosure
-        mainIconPath={VariousIcon}
-        title={`${t("editionsText")} (${manifestations.length})`}
-        disclosureIconExpandAltText=""
-        dataCy="material-editions-disclosure"
-      >
-        <>
-          {getManifestationsOrderByTypeAndYear(manifestations).map(
-            (manifestation: Manifestation) => {
-              return (
-                <MaterialMainfestationItem
-                  key={manifestation.pid}
-                  manifestation={manifestation}
-                  workId={wid}
-                />
-              );
-            }
-          )}
-        </>
-      </Disclosure>
-      <Disclosure
-        mainIconPath={Receipt}
-        title={t("detailsText")}
-        disclosureIconExpandAltText=""
-        dataCy="material-details-disclosure"
-      >
-        <MaterialDetailsList className="pl-80 pb-48" data={detailsListData} />
-      </Disclosure>
-      {reviews && reviews.length >= 1 && (
-        <DisclosureControllable
-          id="reviews"
-          title={t("reviewsText")}
-          mainIconPath={CreateIcon}
-          showContent={shouldOpenReviewDisclosure}
-          cyData="material-reviews-disclosure"
+    <>
+      <CoverPreloadLink id={pid} size="xlarge" />
```

Comment:
Please consider if preloading covers here is actually useful.

From the documentation:

> The preload value of the [<link>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) element's [rel](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-rel) attribute lets you declare fetch requests in the HTML's [<head>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head), specifying resources that your page will need very soon, which you want to start loading early in the page lifecycle, before browsers' main rendering machinery kicks in.

This places the preload link in the HTML body. With that in mind I would not think it made any difference. On the other hand it adds another request (if it is not cached by Orval) and in any case added complexity.

We might be able to move the `<link>` tag to the `<head>` with [help from third party libraries](https://github.com/nfl/react-helmet) but it would render clientside anyway and I do not think it would make much of a difference.

It is also not something we use for the search page which from a performance perspective is even more taxing.

Reply:
Yeah. Think you are right. I don't think the cover preload link here is giving us a noticeable gain. I'll remove it again.